TextToSpeech/impl/TTSManager.cpp

```diff
@@ -189,6 +189,12 @@ TTS_Error TTSManager::setAPIKey(string apikey)
     return TTS_OK;
 }
 
+TTS_Error TTSManager::setSpeechRate(const string speechrate)
+{
+    m_needsConfigStoreUpdate |= m_defaultConfiguration.setSpeechRate(speechrate);
```

Comment:
This is another case, where ITextToSpeech::SetSpeechRate() invocation order depends on ITextToSpeech::SetConfiguration() to properly update the underlying persistent storage (JSON-RPC case works fine due to the management done in that implementation)